Calibration/HcalIsolatedTrackReco/src/EcalIsolatedParticleCandidateProducer.cc

```diff
@@ -211,8 +211,8 @@ EcalIsolatedParticleCandidateProducer::produce(edm::Event& iEvent, const edm::Ev
   // is put into the Event
   
 //  std::cout<<"put cand into event"<<std::endl;
-  std::auto_ptr<reco::IsolatedPixelTrackCandidateCollection> pOut(iptcCollection);
-  iEvent.put(pOut);
+  std::unique_ptr<reco::IsolatedPixelTrackCandidateCollection> pOut(iptcCollection);
+  iEvent.put(std::move(pOut));
 
```

Comment:
@davidlange6 Agreed!
Opened a separate issue: https://github.com/cms-sw/cmssw/issues/14411
Regarding the comment on `std::make_unique`, do you consider this as unrelated to this PR, too?